types/d3-sankey/index.d.ts

```diff
@@ -0,0 +1,267 @@
+// Type definitions for D3JS d3-sankey module 0.4
+// Project: https://github.com/d3/d3-sankey/
+// Definitions by: Tom Wanzek <https://github.com/tomwanzek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Last module patch version validated against: 0.4.2
+
+/**
+ * A helper interface as an extension reference for user-provided properties of
+ * nodes and links in the graph, which are not required or calculated by
+ * the Sankey Layout Generator
+ */
+export interface SankeyExtraProperties { [key: string]: any; }
```

Comment:
This interface seems dangerous. There aren't any default properties, so why not just specify the type parameters in e.g., `SankeyNodeMinimal` as just `<N, L>`?

FWIW, I deleted the interface and none of the tests fail.